vllm/worker/worker.py

```diff
@@ -248,31 +248,15 @@ def get_cache_block_size_bytes(self, block_size: int,
                                                 self.parallel_config)
 
 
-def init_distributed_environment(
+def init_distributed_environment_in_worker(
```

Comment:
Nit:
```suggestion
def init_worker_distributed_environment(
```

Reply:
fixed in [3720e97](https://github.com/vllm-project/vllm/pull/3904/commits/3720e97fcc7e4c3368451fbe729663e993202e84)